Steepfile

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+D = Steep::Diagnostic
+
+target :lib do
+  signature "sig"
+
+  check "lib"
+
+  ignore "lib/prism/serialize.rb"
+  ignore "lib/prism/ffi.rb"
+
+  library "ripper"
```

Comment:
I hope this isn't actually necessary, as it should be hidden and only used for testing.

Reply:
See comment in https://github.com/ruby/prism/pull/2101#discussion_r1445805825